dashboard/modules/job/job_manager.py

```diff
@@ -296,7 +296,9 @@ class JobManager:
     goes down.
     """
 
-    JOB_ACTOR_NAME = "_ray_internal_job_actor_{job_id}"
+    JOB_ACTOR_NAME = (
+        f"{ray_constants.RAY_INTERNAL_NAMESPACE_PREFIX}job_actor_" + "{job_id}"
```

Comment:
```suggestion
        f"{ray_constants.RAY_INTERNAL_NAMESPACE_PREFIX}job_actor_" + f"{job_id}"
```

Reply:
This was actually intentional. `JOB_ACTOR_NAME` is a template. I'll rename the variable to have `TEMPLATE` in the name though.